<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>list-task</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="list">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_add">
       <attribute name="title">
        <string>添加</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_add">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>任务名称</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>描述</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>561</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="lineEdit_description">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>561</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget_2">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>90</y>
          <width>296</width>
          <height>218</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>407</y>
          <width>210</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(61, 123, 185);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>400</y>
          <width>631</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="whale_widget"/>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>320</y>
          <width>364</width>
          <height>23</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>开始时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_begin">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>结束时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_end">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2023</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_list">
       <attribute name="title">
        <string>列表</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1090</width>
          <height>741</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1090</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QPushButton" name="pushButton_del">
        <property name="geometry">
         <rect>
          <x>1130</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>1121</x>
          <y>110</y>
          <width>121</width>
          <height>241</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_calendar">
       <attribute name="title">
        <string>日历</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>0</y>
          <width>851</width>
          <height>771</height>
         </rect>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>291</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
       <widget class="QPushButton" name="pushButton_del_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>330</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_description</tabstop>
  <tabstop>dateEdit_begin</tabstop>
  <tabstop>dateEdit_end</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
